app/controllers/auth_controller.rb

```diff
@@ -22,6 +22,10 @@ def callback
     )
 
     render json: result
+  rescue StandardError => e
+    Rollbar.critical('Server Login Error', e)
+    session[:user] = nil
+    render json: { name: e }, status: :unprocessable_entity
```

Comment:
(I think) This will serialise the whole error `e` in the JSON response? Probably best to only pull out what we need from the error and send that in the response to the client, otherwise we may be at risk of exposing some sensitive data from the error object (especially since this catches any error).

Reply:
Have reduced this to just the error.